src/protocols/PhysicalLayer/PhysicalLayer.h

```diff
@@ -55,6 +55,48 @@ union DataRate_t {
   FSKRate_t fsk;
 };
 
+/*!
+  \struct CADScanConfig_t
+  \brief Channel scan configuration interpretation in case LoRa CAD is used
+*/
+struct CADScanConfig_t {
+  /*! \brief Number of symbols to consider signal present */
+  uint8_t symNum;
+  
+  /*! \brief Number of peak detection symbols */
+  uint8_t detPeak;
+  
+  /*! \brief Number of minimum detection symbols */
+  uint8_t detMin;
+  
+  /*! \brief Exit mode after signal detection is complete - module-specific value */
+  uint8_t exitMode;
+  
+  /*! \brief Timeout in milliseconds */
```

Comment:
I would suggest to put this value in microseconds to match the `getTimeOnAir()` function. That would make it easy to plug the expected timeout straight into the Rx timeout value :)

Reply:
Good point, chagned that.